Issue when cloning exist vm/template

Hello! I'm  trying to make a clone from an existing template. But I'm getting this kind of error

It is noteworthy that the virtual machine is created and works great. If it matters then I don't connect an extra drive to the machine

In principle, this does not interfere with local work, since the machine is created as a result. But I'd like to add this to a CI/CD process where errors are not allowed.

```
Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

proxmox_virtual_environment_vm.ubuntu_vm[0]: Creating...
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [10s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [20s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [30s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [40s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [50s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [1m0s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [1m10s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [1m20s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [1m30s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [1m40s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [1m50s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [2m0s elapsed]
proxmox_virtual_environment_vm.ubuntu_vm[0]: Still creating... [2m10s elapsed]
╷
│ Error: Request cancelled
│ 
│   with proxmox_virtual_environment_vm.ubuntu_vm[0],
│   on resource_proxmox_virtual_environment_vm.tf line 1, in resource "proxmox_virtual_environment_vm" "ubuntu_vm":
│    1: resource "proxmox_virtual_environment_vm" "ubuntu_vm" {
│ 
│ The plugin.(*GRPCProvider).ApplyResourceChange request was cancelled.
╵

Stack trace from the terraform-provider-proxmox_v0.17.1 plugin:

panic: interface conversion: interface {} is nil, not string

goroutine 66 [running]:
github.com/bpg/terraform-provider-proxmox/proxmoxtf/resource.vmGetTagsString(0xd9c5e1?)
        /home/skensel/go/src/github.com/bpg/terraform-provider-proxmox-main/proxmoxtf/resource/vm.go:2690 +0x1c5
github.com/bpg/terraform-provider-proxmox/proxmoxtf/resource.vmCreateClone({0xed93c8, 0xc0004cd1a0}, 0x141d520?, {0xccd9a0?, 0xc000310320?})
        /home/skensel/go/src/github.com/bpg/terraform-provider-proxmox-main/proxmoxtf/resource/vm.go:1637 +0x2827
github.com/bpg/terraform-provider-proxmox/proxmoxtf/resource.vmCreate({0xed93c8, 0xc0004cd1a0}, 0x0?, {0xccd9a0, 0xc000310320})
        /home/skensel/go/src/github.com/bpg/terraform-provider-proxmox-main/proxmoxtf/resource/vm.go:1256 +0x76
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc000488fc0, {0xed9400, 0xc000520d20}, 0xd?, {0xccd9a0, 0xc000310320})
        /home/skensel/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/schema/resource.go:707 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc000488fc0, {0xed9400, 0xc000520d20}, 0xc0001456c0, 0xc00023ed80, {0xccd9a0, 0xc000310320})
        /home/skensel/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/schema/resource.go:837 +0xa85
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc000456060, {0xed9400?, 0xc000520c00?}, 0xc00052e370)
        /home/skensel/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/schema/grpc_provider.go:1021 +0xe8d
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc0002cf5e0, {0xed9400?, 0xc0005201e0?}, 0xc000610070)
        /home/skensel/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/tf5server/server.go:818 +0x574
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0xd6a100?, 0xc0002cf5e0}, {0xed9400, 0xc0005201e0}, 0xc000610000, 0x0)
        /home/skensel/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:385 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0002c21e0, {0xedd378, 0xc000007520}, 0xc000608000, 0xc00047ebd0, 0x140f100, 0x0)
        /home/skensel/go/pkg/mod/google.golang.org/grpc@v1.53.0/server.go:1336 +0xd33
google.golang.org/grpc.(*Server).handleStream(0xc0002c21e0, {0xedd378, 0xc000007520}, 0xc000608000, 0x0)
        /home/skensel/go/pkg/mod/google.golang.org/grpc@v1.53.0/server.go:1704 +0xa36
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        /home/skensel/go/pkg/mod/google.golang.org/grpc@v1.53.0/server.go:965 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /home/skensel/go/pkg/mod/google.golang.org/grpc@v1.53.0/server.go:963 +0x28a

Error: The terraform-provider-proxmox_v0.17.1 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

--- a/proxmoxtf/resource/container.go
+++ b/proxmoxtf/resource/container.go
@@ -605,10 +605,13 @@ func Container() *schema.Resource {
 				},
 			},
 			mkResourceVirtualEnvironmentContainerTags: {
-				Type:                  schema.TypeList,
-				Description:           "Tags of the container. This is only meta information.",
-				Optional:              true,
-				Elem:                  &schema.Schema{Type: schema.TypeString},
+				Type:        schema.TypeList,
+				Description: "Tags of the container. This is only meta information.",
+				Optional:    true,
+				Elem: &schema.Schema{
+					Type:         schema.TypeString,
+					ValidateFunc: validation.StringIsNotEmpty,
+				},
 				DiffSuppressFunc:      suppressIfListsAreEqualIgnoringOrder,
 				DiffSuppressOnRefresh: true,
 			},

--- a/proxmoxtf/resource/vm.go
+++ b/proxmoxtf/resource/vm.go
@@ -1117,10 +1117,13 @@ func VM() *schema.Resource {
 				Default:     dvResourceVirtualEnvironmentVMTabletDevice,
 			},
 			mkResourceVirtualEnvironmentVMTags: {
-				Type:                  schema.TypeList,
-				Description:           "Tags of the virtual machine. This is only meta information.",
-				Optional:              true,
-				Elem:                  &schema.Schema{Type: schema.TypeString},
+				Type:        schema.TypeList,
+				Description: "Tags of the virtual machine. This is only meta information.",
+				Optional:    true,
+				Elem: &schema.Schema{
+					Type:         schema.TypeString,
+					ValidateFunc: validation.StringIsNotEmpty,
+				},
 				DiffSuppressFunc:      suppressIfListsAreEqualIgnoringOrder,
 				DiffSuppressOnRefresh: true,
 			},

